### PR TITLE
add support for nix-shell

### PR DESCRIPTION
**Pull request by gvolpe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,22 @@ Not yet, but flake.lock makes it reproducible.
 
 ###	Can I use it with trusty old `nix-shell`?
 
-Probably, but I haven't put time into how.  I'm leaning into flakes.
+Absolutely! The `shell.nix` provides a flakes-compatible shell that works with `nix-shell`. It selects the `application` shell by default, but you can be specific about it. E.g.
```
Comment (rossabaker) [inline README.md:74]: I wondered whether there should be a default devShell as well.

**Review by rossabaker — APPROVED** [review]
Nice, thank you.

File: README.md (hunk 1 above)
Comment (gvolpe) [reply README.md:74]: I guess it's a matter of deciding what "default" means :)

Comment (rossabaker) [reply README.md:74]: Yep, exactly.  I mostly envision this being embedded in projects with direnv, but what we have already is a nice one-liner for unnixxed projects.

**Pull request merged**